src/css_composer/view/CssRulesView.js

```diff
@@ -61,19 +62,41 @@ module.exports = require('backbone').View.extend({
       rendered = view.render().el;
     }
 
+    const mediaWidth = this.getMediaWidth(model.get('mediaText'));
+    const styleBlockId = `${this.pfx}rules-${mediaWidth}`;
+
     if (rendered) {
-      if (fragment) fragment.appendChild(rendered);
-      else this.$el.append(rendered);
+      if (fragment) {
+        fragment.getElementById(styleBlockId).appendChild(rendered);
+      } else {
+        let $stylesContainer = this.$el.find(`#${styleBlockId}`);
+        $stylesContainer.append(rendered);
+      }
     }
 
     return rendered;
   },
 
+  getMediaWidth(mediaText) {
+    return mediaText && mediaText.replace('(max-width: ', '').replace(')', '');
+  },
+
   render() {
     this.atRules = {};
     const $el = this.$el;
     const frag = document.createDocumentFragment();
     $el.empty();
+
+    // Create devices related DOM structure
+    const pfx = this.pfx;
+    this.em
+      .get('DeviceManager')
+      .getAll()
+      .forEach(function(model) {
+        const blockId = pfx + 'rules-' + model.get('widthMedia');
+        $(`<div id="${blockId}"></div>`).appendTo(frag);
+      });
+
```

Comment:
It will work only when devices are already sorted by `widthMedia` which is not always true. Might be a case to use `sortBy` method:
```js
.getAll()
.sortBy(device => ...)
.forEach
```